backend/clubs/models.py

```diff
@@ -965,6 +965,15 @@ def __str__(self):
         )
 
 
+class Search(models.Model):
```

Comment:
You might want to rename this to something more descriptive, like `SearchQuery`.

Reply:
Good call